todo_app/app.py

```diff
@@ -61,6 +63,4 @@ def healthcheck():
 
     return app
 
-if __name__ == '__main__':
-    app = create_app()
-    app.run()
+application = create_app()
```

Comment:
Not a problem, but does anything actually use this? You should be fine to get rid of it entirely since Gunicorn and Flask are both happy using the create_app function directly

Reply:
Have made the change you suggested, but have to say I'm quite confused. I thought a previous version of this app had failed without this. I could probably do with a more detailed explanation of how Python works here, any useful docs on the logic around start-up of apps in Python is appreciated.